### DISCUSSION TITLE
Resize taskbar after moving

### DISCUSSION THREAD

**Discussion by bgwartney** [kangyu-california/PersistentWindows#141] [Ideas]
Thanks  so much for this great time saving tool.

I run my taskbar on different monitor, and have it dragged up larger than the default single line. When PW moves the taskbar back to the correct display, it does not resize it back to it's previous size. It'd be a really nice feature for it to also resize the taskbar to what it was when the state was saved.

**Comment by kangyu-california** [comment]
Converted to issue #142
